src/table/head/table-head-cell.component.ts

```diff
@@ -17,22 +17,21 @@ import { TableHeaderItem } from "./../table-header-item.class";
 	template: `
 		<button
 			class="bx--table-sort"
-			*ngIf="sortable && !skeleton && this.sort.observers.length > 0 && column.sortable"
+			*ngIf="sortable && this.sort.observers.length > 0 && column.sortable"
 			[attr.aria-label]="(column.sorted && column.ascending ? getSortDescendingLabel() : getSortAscendingLabel()) | async"
 			aria-live="polite"
 			[ngClass]="{
 				'bx--table-sort--active': column.sorted,
 				'bx--table-sort--ascending': column.ascending
 			}"
 			(click)="onClick()">
-			<span
-				*ngIf="!column.template"
+			<div
```

Comment:
I need it to set the `tabindex`, otherwise I get the following error:
```
Cannot set property 'tabIndex' of undefined
    at Function.push../src/table/table.component.ts.Table.setTabIndex (table.component.ts:330)
```

Reply:
Who calls that? grid interactions? Consider adding a comment explaining that.